### PR TITLE
Add list of default values to change to secure install

### PR DESCRIPTION
**Pull request by alhambrav**
### Ticket reference or full description of what's in the PR
Add list of default values to change to secure install

### CODE REVIEW

File: source/system-administrators/activities/security/security-change-defaults.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        export STUDIO_MANAGEMENT_TOKEN=${STUDIO_MANAGEMENT_TOKEN:="defaultManagementToken"}
+        export ENGINE_MANAGEMENT_TOKEN=${ENGINE_MANAGEMENT_TOKEN:="defaultManagementToken"}
+        export DEPLOYER_MANAGEMENT_TOKEN=${DEPLOYER_MANAGEMENT_TOKEN:="defaultManagementToken"}
+        export SEARCH_MANAGEMENT_TOKEN=${SEARCH_MANAGEMENT_TOKEN:="defaultManagementToken"}
+        export PROFILE_MANAGEMENT_TOKEN=${PROFILE_MANAGEMENT_TOKEN:="defaultManagementToken"}/*
+        export SOCIAL_MANAGEMENT_TOKEN=${SOCIAL_MANAGEMENT_TOKEN:="defaultManagementToken"}
+
+     |
+
+* Replace default values for the DB root password and the DB ``crafter`` user password
```
Comment (avasquez614) [inline source/system-administrators/activities/security/security-change-defaults.rst:42]: I think we need to indicate that these values can't be changed after the DB is installed (if using the embedded DB). agree @dejan-brkic?

File: source/system-administrators/activities/security/security-change-defaults.rst [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+        # -------------------- MariaDB variables --------------------
+        ...
+        export MARIADB_ROOT_PASSWD=${MARIADB_ROOT_PASSWD:="root"}
+        ...
+        export MARIADB_PASSWD=${MARIADB_PASSWD:="crafter"}
+
+     |
+
+* Replace the default values for the cipher key and salt in Studio
```
Comment (avasquez614) [inline source/system-administrators/activities/security/security-change-defaults.rst:55]: I think we need to indicate that these values can't be changed after the DB is installed. agree @dejan-brkic?

File: source/system-administrators/activities/security/security-change-defaults.rst (hunk 1 above)
Comment (dejan-brkic) [reply source/system-administrators/activities/security/security-change-defaults.rst:42]: agree

File: source/system-administrators/activities/security/security-change-defaults.rst (hunk 2 above)
Comment (dejan-brkic) [reply source/system-administrators/activities/security/security-change-defaults.rst:55]: agree

File: source/system-administrators/activities/security/security-change-defaults.rst [hunk 3]
```diff
@@ -39,7 +39,7 @@ Crafter CMS installations are pre-configured with default passwords, tokens, key
 
      |
 
-* Replace default values for the DB root password and the DB ``crafter`` user password
+* Replace default values for the DB root password and the DB ``crafter`` user password before starting Crafter CMS.  Remember that these values should **not** be changed after the DB has been installed
```
Comment (sumerjabri) [inline source/system-administrators/activities/security/security-change-defaults.rst:42]: What happens if they want to change after they start (for example the AWS AMI product line)?
Perhaps we can explain that if you want to change after initial start, you must do XYZ.

**Pull request merged**